src/components/PantallaValidacion/index.module.scss

```
@@ -19,8 +19,12 @@
 
 }
 
-.button{
+.buttons{
     display: flex;
     flex-direction: row;
     justify-content: center;
 }
+
+.inferior {
+    transform: scaleY(-1);
+}
```

Ojo con la identación de este file (pasalo todo a 2 espacios):
```suggestion
.inferior {
  transform: scaleY(-1);
}
```